frontend/packages/console-app/src/components/user-preferences/UserPreferences.tsx

```diff
@@ -0,0 +1,137 @@
+import { getUserSettingsGroups } from './utils/getUserSettingsGroups';
+
+export type UserPreferencesProps = RouteComponentProps<{ category: string }>;
+
+const UserPreferences: React.FC<UserPreferencesProps> = ({ match }) => {
+  // resources and calls to hooks
+  const { t } = useTranslation();
+
+  const [userSettingsExtensions, userSettingsResolved] = useResolvedExtensions<UserSettings>(
+    isUserSettings,
+  );
+  const sortedUserSettingsExtensions = orderExtensionBasedOnInsertBeforeAndAfter<
+    ResolvedUserSettings
+  >(userSettingsExtensions.map(({ properties }) => properties));
+
+  const userSettingsGroupExtensions = useExtensions<UserSettingsGroup>(isUserSettingsGroup);
+  const sortedUserSettingsGroupExtensions = orderExtensionBasedOnInsertBeforeAndAfter<
+    ResolvedUserSettingsGroup
+  >(userSettingsGroupExtensions.map(({ properties }) => properties));
+
+  // fetch the default category from the url params if available
+  const {
+    url,
+    params: { category: groupIdFromUrl },
+  } = match;
+  const { id: initialTabId } =
+    sortedUserSettingsGroupExtensions.length > 0
+      ? sortedUserSettingsGroupExtensions.find((extension) => extension.id === groupIdFromUrl) ||
+        sortedUserSettingsGroupExtensions[0]
+      : { id: '' };
+  const [activeTabId, setActiveTabId] = React.useState<string>(initialTabId);
+
+  // utils and callbacks
+  const getTabs = (): [React.ReactElement<TabProps>[], React.ReactElement<TabContentProps>[]] => {
+    const userSettingsGroup: UserSettingsTabGroup[] = getUserSettingsGroups(
+      sortedUserSettingsExtensions,
+      sortedUserSettingsGroupExtensions,
+    );
+    const [tabs, tabContents] = userSettingsGroup.reduce(
+      (acc, currGroup) => {
+        const { id, label, items } = currGroup;
+        const ref = React.createRef<HTMLElement>();
+        acc[0].push(
+          <Tab
+            key={id}
+            eventKey={id}
+            title={<TabTitleText>{label}</TabTitleText>}
+            tabContentId={id}
+            tabContentRef={ref}
+            data-test={`userPreferenceTab ${id}`}
+          />,
+        );
+        acc[1].push(
+          <TabContent
+            key={id}
+            eventKey={id}
+            id={id}
+            ref={ref}
+            hidden={id !== activeTabId}
+            data-test={`userPreferenceTabContent ${id}`}
+          >
+            <Form>{getTabContents(items)}</Form>
+          </TabContent>,
+        );
+        return acc;
+      },
+      [[], []],
+    );
+    return [tabs, tabContents];
+  };
+  const handleTabClick = (event, eventKey) => {
+    setActiveTabId(eventKey);
+    history.replace(`${url}/${eventKey}`);
+  };
+
+  // render elements
+  const [userSettingsTabs, userSettingsTabContents] = getTabs();
+
+  return (
+    <>
+      <Helmet>
+        <title>{t('console-app~User Settings')}</title>
+      </Helmet>
+      <PageHeading detail title={t('console-app~User Settings')} />
```

Comment:
The design document says the title should be "User preferences". There is also an subtitle below the page title in the design.

From design doc:

![image](https://user-images.githubusercontent.com/139310/126467817-8fbfb4f1-d3ea-450d-b81c-eb6ff6f3005e.png)